heartbeat/docs/monitors/monitor-browser.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[monitor-browser-options]]
+=== Browser options
+
+Also see <<monitor-options>>.
+
+The options described here configure {beatname_uc} to run the synthetic
+monitoring test suites via Synthetic Agent on the chromium browser.
+
+Example configuration:
+
+[source,yaml]
+----
+- type: browser
+  id: synthetic-inline-suites
+  name: Elastic website
+  schedule: '@every 1m'
+  source:
+    inline:
+      script: |-
+        step("load homepage", async () => {
+          await page.goto('https://www.elastic.co');
+        });
+        step("hover over products menu", async () => {
+          await page.hover('css=[data-nav-item=products]');
+        });
+----
+
+[float]
+[[monitor-browser-source]]
+==== `Source`
+
+Details about the how to run the synthetic test suites.
+
+[float]
+[[monitor-source-inline]]
+==== `Inline`
+
+Runs the Synthetic test scripts that are defined inline in the configuration.
+See <<synthetics-syntax>> for more information.
+
+
+[float]
+[[monitor-source-zipendpoint]]
+==== `Zip endpoint`
```

Comment:
Why is this titled `zip endpoint` instead of `zip_url`? Also, we should nest this under `source`
```suggestion
===== `Zip endpoint`
```